app/controllers/public_pages_controller.rb

```diff
@@ -40,4 +40,8 @@ def stimulus_recommendation; end
   def sms_terms; end
 
   def diy; end
+
+  def faq; end
```

Comment:
oh, not sure where this came from - must have already been there when i took over the story

Reply:
Ya @caitbot and I added that. It seemed like it didn't need to be defined (I assume rails implicitly creates it since there's a view) and it still worked. But we wanted to be explicit so that folks can navigate from the controller to the template.